pcsx2/GS/GSState.cpp

```diff
@@ -3111,34 +3111,34 @@ __forceinline void GSState::VertexKick(u32 skip)
 
 	// callers should write XYZUVF to m_v.m[1] in one piece to have this load store-forwarded, either by the cpu or the compiler when this function is inlined
 
-	GSVector4i v0(m_v.m[0]);
-	GSVector4i v1(m_v.m[1]);
+	GSVector4i tail_v0(m_v.m[0]);
+	GSVector4i tail_v1(m_v.m[1]);
```

Comment:
they aren't the tail, they're going in to the the tail (further down), they are the new vertex information

Reply:
`new_v0/1` then, wanted to avoid having them as duplicates with the ones a bit down.

Reply:
sure